src/client/Microsoft.Identity.Client/ApiConfig/Executors/ConfidentialClientExecutor.cs

```diff
@@ -37,7 +37,7 @@ public ConfidentialClientExecutor(IServiceBundle serviceBundle, ConfidentialClie
                 commonParameters,
                 requestContext,
                 _confidentialClientApplication.UserTokenCacheInternal).ConfigureAwait(false);
-            requestParams.SendX5C = authorizationCodeParameters.SendX5C;
+            requestParams.SendX5C = authorizationCodeParameters.SendX5C ?? false;
```

Comment:
Wonder if this `false` can be a default constant like here https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/351cbba17d6d34fe1d10fae2128a7df04eee3a9d/src/client/Microsoft.Identity.Client/AppConfig/ApplicationConfiguration.cs#L20

Reply:
I do not think that we should make a default const for a bool esp. if there are no other values (other than true).